src/browser-frontend/views/browser/window.jsx

```diff
@@ -33,19 +32,20 @@ import Content from './content';
 })
 export default class Window extends PureComponent {
   componentDidMount() {
+    // Some keyboard shortcuts need to be registered at the platform level,
+    // because they clash with OS-level shortcuts (e.g. Cmd+W on macOS
+    // hides windows, but we want it to close tabs instead.). The platform
+    // is tasked with overriding these shortcuts and providing us control.
```

Comment:
I do wonder a little about having two different places to configure key events.  Would it be terrible to put `CommandOrControl+T` and the Konami code in the platform, too?  Or move them to a saga?